pkg/accesscontrol/access_store.go

```diff
@@ -63,6 +64,10 @@ func (l *AccessStore) AccessFor(user user.Info) *AccessSet {
 	return result
 }
 
+func (l *AccessStore) PurgeUserData(id string) {
```

Comment:
What does this other access store cache store and why? And why is its size 50?

Reply:
This caches a key representing the aggregate permissions given to a user to the access set created for that set of permissions. We expose a purge method for this so that when we have determined that a user's permissions have changed and that we should no longer cache the old permissions, we can purge that from the cache here as well.

I have no idea why this is size 50 - I can't think of a discernible reason for that number as opposed to another number.